pypeit/setup_gui/view.py

```diff
@@ -1317,6 +1586,15 @@ def _logClosed(self):
         """Signal handler that clears the log window when it closes."""
         self._logWindow = None
             
+    def _helpButton(self):
+        """Signal handler that responds to the help button being pressed."""
+
+        result = QDesktopServices.openUrl(QUrl("https://pypeit.readthedocs.io/en/latest/"))
```

Comment:
Can this instead point to `https://pypeit.readthedocs.io/en/stable/setup.html`?  It would also be good to add some GUI-specific instructions there.